### PR TITLE
Add documentation for building on Windows

### PR DESCRIPTION
**Pull request by wtgodbe**
This is just a first crack, and is very similar to the corresponding document in CoreCLR. Someone more familiar with this repo may want to expand on this, and/or add a similar document for Unix/OSX.

CC @gkhanna79 @Petermarcu @eerhardt

### CODE REVIEW

**Comment by gkhanna79** [comment]
Thanks for doing this! You probably want to use this oppurtunity to add corresponding OSX and Linux info as well.

**Comment by wtgodbe** [comment]
@gkhanna79 I don't know what the prereqs are there, do you know/know who would?

**Comment by rakeshsinghranchi** [comment]
@wtgodbe  : 
Few weeks back when I tried on a clean Win10 machine to build , noted down these pre-requisites : 
1. 	Visual Studio 2015 Update 3 with VC++ Under Visual C++ select "Common Tools for Visual C++ 2015"
2. 	Windows SDK 8.1 https://developer.microsoft.com/en-us/windows/downloads/windows-8-1-sdk
3.     Cmake https://cmake.org/download/

File: Documentation/building/windows-instructions.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+PowerShell is used in the build system. Ensure that it is accessible via the PATH environment variable.
+Typically this is %SYSTEMROOT%\System32\WindowsPowerShell\v1.0\.
+
+Powershell version must be 3.0 or higher. This should be the case for Windows 8 and later builds.
+- Windows 7 SP1 can install Powershell version 4 [here](https://www.microsoft.com/en-us/download/details.aspx?id=40855).
+
+## DotNet Core SDK
+While not strictly needed to build or tests the .NET Core repository, having the .NET Core SDK installed lets 
+you use the dotnet.exe command to run .NET Core applications in the 'normal' way.   We use this in the 
+[Using Your Build](Documentation/workflow/UsingYourBuild.md) instructions.  Visual Studio 2015 (update 3) should have
```
Comment (eerhardt) [inline Documentation/building/windows-instructions.md]: Busted link.

File: Documentation/building/windows-instructions.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+installed the .NET Core SDK, but in case it did not you can get it from the [Installing the .Net Core SDK](https://www.microsoft.com/net/core) page.  
+
+## Adding to the default PATH variable
```
Comment (eerhardt) [inline Documentation/building/windows-instructions.md]: Is this necessary?

**Pull request merged**